blockchain/blocks/src/header.rs

```diff
@@ -396,7 +396,7 @@ mod tests {
         assert_eq!(&header_bytes, &HEADER_BYTES)
     }
 
-    fn build_header() -> BlockHeader {
+    pub fn build_header() -> BlockHeader {
```

Comment:
doesn't matter because it's in the unexported module, but why this change?

Reply:
I was using it while I was writing unit tests, will revert back until then.